addons/controllers/suite_test.go

```diff
@@ -121,7 +122,7 @@ var _ = BeforeSuite(func(done Done) {
 	Expect(externalCRDPaths).ToNot(BeEmpty())
 	testEnv.CRDDirectoryPaths = externalCRDPaths
 	testEnv.CRDDirectoryPaths = append(testEnv.CRDDirectoryPaths,
-		filepath.Join("..", "..", "config", "crd", "bases"), filepath.Join("testdata"))
+		filepath.Join("..", "..", "config", "crd", "bases"), filepath.Join("testdata"), filepath.Join("testdata", "dependency", "crd"))
```

Comment:
Should we get the crd from the [repo itself](https://github.com/vmware-tanzu/vm-operator/tree/main/config/crd/bases) like we are doing for other external crds for consistency?

Reply:
Including vm-operator will introduce many dependency complications in go.mod. This is the reason they introduced `vm-operator-api`, but unfortunately, it doesn't include CRD files.

Reply:
Ah, more trouble than actually worth it. Could you add a code comment with this last bit of explanation that vm-operator-api causes depedency conflicts.